cypress/TestCases/FireboltCertification/Device.feature

```diff
@@ -67,3 +67,59 @@ Feature: Device
             | Validate network_Ethernet_disconnected | device network as ethernet disconnected | onNetworkChanged with ethernet disconnected | onNetworkChanged events with ethernet disconnected |
             | Validate network_Hybrid_connected      | device network as hybrid connected      | onNetworkChanged with hybrid connected      | onNetworkChanged events with hybrid connected      |
             | Validate network_Hybrid_disconnected   | device network as hybrid disconnected   | onNetworkChanged with hybrid disconnected   | onNetworkChanged events with hybrid disconnected   |
+
+    @Device @mfos @coreSDK @regression @sdk
+    Scenario Outline: Device.<Method> - Positive Scenario: <Scenario>
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with '<validation_key>'
+
```

Comment:
Device.audioFormatSupported - Positive Scenario: Validate audioFormatSupported 
Device.audioFormatSupported - Positive Scenario: Validate videoFormatSupported 

1. For these test cases, I could see only one example passed as params. But AudioCodec and VideoCodec supports other params too . Do we have to add them as additional TC ? 

"AudioCodec": {
				"type": "string",
				"description": "Audio codec",
				"enum": [
					"aac",
					"ac3",
					"ac4",
					"dts-x",
					"eac3",
					"mpeg3",
					"opus",
					"truehd",
					"unknown",
					"vorbis"
				]
			},
			"VideoCodec": {
				"type": "string",
				"description": "video codec",
				"enum": [
					"av1",
					"avc",
					"hevc",
					"mpeg1",
					"mpeg2",
					"vp8",
					"vp9",
					"vp10"
				]
			},


2. Also, what if we pass any other key/value as params ? do we have TC for that

Reply:
Added additional testcases for above